server/routerlicious/packages/services/src/containerManager.ts

```diff
@@ -0,0 +1,50 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ScopeType } from "@fluidframework/protocol-definitions";
+import { BasicRestWrapper } from "@fluidframework/server-services-client";
+import { IContainerManager, IDocument } from "@fluidframework/server-services-core";
+import { generateToken, getCorrelationId } from "@fluidframework/server-services-utils";
+
+/**
+ * Manager to fetch container from Alfred using the internal URL.
+ */
+export class ContainerManager implements IContainerManager {
```

Comment:
Where would this be used?

Reply:
See this comment : https://github.com/microsoft/FluidFramework/pull/15190#discussion_r1172591293